connectors/src/api/webhooks/webhook_firecrawl.ts

```diff
@@ -0,0 +1,86 @@
+import { assertNever } from "@dust-tt/client";
+import type { Request, Response } from "express";
+
+import mainLogger from "@connectors/logger/logger";
+import { withLogging } from "@connectors/logger/withlogging";
+import type { WithConnectorsAPIErrorReponse } from "@connectors/types";
+
+const logger = mainLogger.child({
+  provider: "webcrawler",
+  service: "firecrawl",
+});
```

Comment:
fyi for later, we can add `{ msgPrefix: "[Firecrawl] " }` as a second option of the child logger to avoid writing every where and typo.